doc/citing.rst

```diff
@@ -3,8 +3,12 @@
 ================
 
 If HyperSpy has been significant to a project that leads to an academic
-publication, please acknowledge that fact by citing the software. Click on
-the DOI badge below for formatted citation formats.
+publication, please acknowledge that fact by citing it. The DOI in the
+badge below is the Concept DOI of HyperSpy. It can be used to cite the project
```

Comment:
`Concept DOI (for all versions)` because it isn't obvious.
Add link to https://help.zenodo.org ?

Reply:
Done